.github/actions/install-arches-applications/action.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Install Arches Applications'
```

Comment:
What do you think about renaming this to something like Install Private Arches Applications? The description right below reads very clearly, explaining you probably don't need this if you've declared a (public) python dependency in the usual way.

Reply:
Eh I'm unsure, because it's not always private. Let's say you have an application that you've hosted on github but not pypi. You could feasibly use this pattern instead of linking in `requirements.txt`. I agree there needs to be better signposting though